rpxdock/sampling/xform_hier.py

```diff
@@ -64,7 +63,7 @@ def hier_axis_sampler(
    else:
       samp = rp.sampling.RotCart1Hier_f4(lb, ub, cart_nstep, 0,
                                       ang, ang_nstep, axis[:3])
-   if kw.flip_components[0]:
+   if kw.flip_components != None:
```

Comment:
I think `if kw.flip_components != None:` will always evaluate to False when running rpxdock through dock.py, but it can be None when running pytest. This should go back to if `kw.flip_comments[0]:` and edit the test to handle the None case I think.